resources/assets/uad_lists.json

```
@@ -9461,7 +9461,7 @@
   {
     "id": "com.mediatek.dataprotection",
     "list": "Misc",
-    "description": "It have stuff about encrypt, decrypt user partition.\nMy device works flawlesly without it.",
+    "description": "Possibly related to user partition encryption / decryption.\nA device should works flawlesly without it.",
```

```suggestion
    "description": "Possibly related to user partition encryption/decryption.\nA device should works flawlessly without it.",
```

In my opinion the removal should be `Advanced`, as it breaks encryption of the partition. That's a huge security flaw if it's not encrypted imo.